update status correction docs

roxanne-moslehi opened the pull request:
<!-- *Note: Please remember to review the Datadog Documentation [Contribution Guidelines](https://github.com/DataDog/documentation/blob/master/CONTRIBUTING.md) if you have not yet done so.* -->

### What does this PR do? What is the motivation?
<!-- A brief description of the change being made with this pull request. What is your motivation for the PR? -->

### Merge instructions
<!-- If you want us to merge this PR as soon as we've reviewed, check the box below. If you're waiting for a release or there are other considerations that you want us to be aware of, list them below. -->

- [ ] Please merge after reviewing

### Additional notes
<!-- Anything else we should know when reviewing?-->

<!-- Previewing the PR: Assuming you are a Datadog employee and named your branch `<yourname>/<description>`, a preview build will run and links to the preview output will be auto-generated and posted in the PR comments. The links will 404 until the preview build is finished running. -->

roxanne-moslehi commented:
FYI @mmeyer724 - also I'm going to remove the note that the data source is in public beta?

@domalessi this is ready to review but we will hold off on merging until we have released the feature

estherk15 commented on static/images/dashboards/guide/slo_data_type/slo-data-source-correction.gif:
This needs to be a mp4 file

estherk15 commented on static/images/service_management/service_level_objectives/slo-correction-toggle.gif:
Instead of a video file, can you take a screenshot of this as a PNG. Make sure to remove/blur your name for PII.

estherk15 commented on content/en/dashboards/guide/slo_data_source.md:
```diff
@@ -34,6 +34,11 @@ For the *Measure* parameter, see the table below for more information on what ea
 
+#### SLO status corrections
+
+For SLOs that have [status corrections][2], you can decide whether you want to enable or disable corrections in charts using the SLO data source. By default, corrections will be enabled and applied to the chart. You can update the behavior for displaying corrections via the advanced options menu options:
```
```suggestion
For SLOs that have [status corrections][2], you can enable or disable corrections in charts using the SLO data source. By default, corrections are enabled and applied to the chart; you can disable with the advanced options menu options:
```

estherk15 commented on content/en/dashboards/guide/slo_data_source.md:
```diff
@@ -34,6 +34,11 @@ For the *Measure* parameter, see the table below for more information on what ea
+
+For SLOs that have [status corrections][2], you can decide whether you want to enable or disable corrections in charts using the SLO data source. By default, corrections will be enabled and applied to the chart. You can update the behavior for displaying corrections via the advanced options menu options:
+
+{{< img src="dashboards/guide/slo_data_type/slo-data-source-correction.gif" alt="Graph editor configuration with the slo data type selected and the good events measure selected" style="width:100%;" >}}
```
This needs to be an MP4 file. Let me know if you have any questions on this!

estherk15 commented on content/en/service_management/service_level_objectives/_index.md:
```diff
@@ -304,8 +302,16 @@ To access SLO status corrections in the UI:
 
+#### Visualizing status corrections
+
+For metric-based and Time Slice SLOs with status corrections, there is a toggle in the SLO detail view that lets you enable or disable corrections in the UI. Note that your overall SLO status and error budget will always take status corrections into consideration. The toggle just controls the charts and data in the "History" section of the SLO detail view.
```
Capitalizing for consistency.

```suggestion
For Metric-based and Time Slice SLOs with status corrections, there is a toggle in the SLO detail view that lets you enable or disable corrections in the UI. The toggle controls the charts and data in the "History" section of the SLO detail view. **Note**: Your overall SLO status and error budget always takes status corrections into consideration. 
```

estherk15 commented on content/en/service_management/service_level_objectives/_index.md:
```diff
@@ -304,8 +302,16 @@ To access SLO status corrections in the UI:
+
+For metric-based and Time Slice SLOs with status corrections, there is a toggle in the SLO detail view that lets you enable or disable corrections in the UI. Note that your overall SLO status and error budget will always take status corrections into consideration. The toggle just controls the charts and data in the "History" section of the SLO detail view.
+
+{{< img src="service_management/service_level_objectives/slo-correction-toggle.gif" alt="SLO correction UI" style="width:100%;">}}
```
Recommend an image over a video

estherk15 reviewed (approved):
Looks good, left some suggestions.

roxanne-moslehi commented:
@estherk15 thanks for the comments, do we not use videos in our docs?

estherk15 commented:
@roxanne-moslehi There's a private beta banner on this page. Do you know if this is going to be transitioned to "Preview"?

roxanne-moslehi commented:
@estherk15 addressed your feedback! kept one video and made the other an image
For private beta banners, I can follow up on that separately. We have a monthly PM meeting tomorrow to discuss this change and what we need to do.

roxanne-moslehi commented:
@estherk15 this is good to merge!
